packages/logs/README.md

```diff
@@ -270,8 +270,8 @@ createLogger (name: string, conf?: {
 
 After the creation of a logger, access it in any part of your JavaScript code with the API:
 
-```javascript
-getLogger (name: string)
+```text
```

Comment:
nit: what about using typescript when applicable? syntax coloration could be helpful